app/views/RoomsListView/index.js

```diff
@@ -354,14 +364,16 @@ class RoomsListView extends React.Component {
 			},
 			headerRight: () => (searching ? null : (
 				<CustomHeaderButtons>
-					<Item
-						title='new'
-						iconName='new-chat'
-						onPress={isMasterDetail
-							? () => navigation.navigate('ModalStackNavigator', { screen: 'NewMessageView' })
-							: () => navigation.navigate('NewMessageStackNavigator')}
-						testID='rooms-list-view-create-channel'
-					/>
+					{hasCreateChannelPermission || hasCreateDirectMessagePermission || hasCreatePrivateGroupPermission ? (
```

Comment:
I guess this button shouldn't be hide when we can't create chat's, since the `NewMessageView` is a "Contacts List" too. WDYT?

Reply:
You're right.

To achieve a real Contact List, I've disabled the possibility to enter to the chat when the user has no view permission.